myuw/resources/sws/file/student/v5/enrollment.json_reg_id_10000000000000000000000000000010_verbose_true_transcriptable_course_all_changed_since_date_

```diff
@@ -0,0 +1,312 @@
+                    },
+                    "RepeatCourse": false,
+                    "RepositoryTimeStamp": "3/28/2013 1:16:41 PM",
+                    "RequestDate": "",
+                    "RequestStatus": "",
+                    "Section": {
+                        "CourseNumber": "201",
+                        "CurriculumAbbreviation": "COM",
+                        "Href": "/student/v5/course/2013,winter,COM,201/A.json",
+                        "Quarter": "winter",
+                        "SectionID": "A",
+                        "Year": 2013
+                    },
+                    "StartDate": "01/30/2013",
+                    "VariableCredit": false,
+                    "WritingCourse": false
+                },
+                {
+                    "AccessDateRangeEnd": "07/25/2013",
+                    "AccessDateRangeStart": "12/19/2012",
+                    "Auditor": false,
+                    "Credits": "4.0",
+                    "DuplicateCode": "0",
+                    "EducationUnitType": null,
+                    "EndDate": "04/30/2013",
+                    "FeeBaseType": "",
+                    "Grade": "X",
+                    "GradeDate": "",
+                    "GradeDocumentID": "",
+                    "GradingSystem": null,
+                    "HonorsCourse": false,
+                    "Href": "/student/v5/registration/2013,winter,PSYCH,203,A,27844DAE1712511A4FA00C83BAF90A8F,0.json",
+                    "Instructor": null,
+                    "IsActive": true,
+                    "IsCredit": true,
+                    "IsIndependentStart": false,
+                    "Metadata": "RegistrationSourceLocation=SDB;",
+                    "Person": {
+                        "Href": "/student/v5/person/27844DAE1712511A4FA00C83BAF90A8F.json",
+                        "Name": "Jane Pce",
+                        "RegID": "27844DAE1712511A4FA00C83BAF90A8F"
+                    },
+                    "RepeatCourse": false,
+                    "RepositoryTimeStamp": "3/28/2013 1:17:37 PM",
+                    "RequestDate": "",
+                    "RequestStatus": "",
+                    "Section": {
+                        "CourseNumber": "203",
+                        "CurriculumAbbreviation": "PSYCH",
+                        "Href": "/student/v5/course/2013,winter,PSYCH,203/A.json",
+                        "Quarter": "winter",
+                        "SectionID": "A",
+                        "Year": 2013
+                    },
+                    "StartDate": "01/29/2013",
+                    "VariableCredit": false,
+                    "WritingCourse": false
+                }
+            ],
+            "RepositoryTimeStamp": "3/28/2013 2:22:05 PM",
+            "Term": {
+                "Href": "/student/v5/term/2013,winter.json",
+                "Quarter": "winter",
+                "Year": 2013
+            }
+        }
```

Comment:
line 138 missed a comma, which is fixed on feature/purple branch. 
https://github.com/uw-it-aca/myuw/pull/595